assets/src/edit-story/components/library/panes/media/deleteDialog.js

```diff
@@ -96,19 +82,17 @@ function DeleteDialog({ mediaId, type, onClose }) {
     <Dialog
       open={true}
       onClose={onClose}
-      title={type == 'image' ? imageDialogTitle : videoDialogTitle}
+      title={type === 'image' ? imageDialogTitle : videoDialogTitle}
```

Comment:
These strings could probably be using `sprintf` instead of duplicating all the text except for the type? Out of scope for this PR though :)

Reply:
Nope :-)

In some languages the translation could be different depending on whether it's an image or a video. That's why it makes sense to keep those separate.

Reply:
Aha, interesting. Alright :)